### ISSUE TITLE
Import from Logseq

### ISSUE THREAD

**Issue by NishantTharani** [obsidianmd/obsidian-importer#47]
I think there's a market for an importer from Logseq to Obsidian.

I've made available this Python script that I used to convert my own Logseq notes to Obsidian - https://github.com/NishantTharani/LogSeqToObsidian - someone could find it useful as a basis for an importer. (Or not!)

**Comment by glaforge** [comment]
I didn't succeed converting my Logseq notes with the python script in the repository mentioned.
However, I was able to export to Roam JSON format first, from Logseq, and import that in Obsidian with the Roam import ability.

**Comment by davidojedalopez** [comment]
> I didn't succeed converting my Logseq notes with the python script in the repository mentioned. However, I was able to export to Roam JSON format first, from Logseq, and import that in Obsidian with the Roam import ability.

I tried this and got a weird export out of LogSeq. The JSON file has the correct title of my pages, but the content of each page is just a random, repeated string like `QA5XlXtROtQA5XlXtROyQA5XlXtROpQA5XlXtROeQA5XlXtRO`.

**Comment by minimaldesign** [comment]
> > I didn't succeed converting my Logseq notes with the python script in the repository mentioned. However, I was able to export to Roam JSON format first, from Logseq, and import that in Obsidian with the Roam import ability.
> 
> I tried this and got a weird export out of LogSeq. The JSON file has the correct title of my pages, but the content of each page is just a random, repeated string like `QA5XlXtROtQA5XlXtROyQA5XlXtROpQA5XlXtROeQA5XlXtRO`.

Same exact experience.

The python script just was too far off in its output to be a workable solution unfortunately…

**Comment by BartVB** [comment]
I used the script @NishantTharani provided. Only issue that I had is that it was unable to copy some attachements but I can do without. Didn't bother to debug/fix that.

Don't have a complicated LogSeq database, but for creating readable/workable notes and Journal entries this was perfect!

**Comment by kepano** [comment]
A new option is available here:
https://github.com/sercxanto/logseq_to_obsidian

**Comment by builder555** [comment]
after converting from logseq to obsidian using [LogSeqToObsidian](https://github.com/NishantTharani/LogSeqToObsidian) i just got 100s of disconnected journals. obsidian doesn't link tags the way logseq does?

**Comment by fschuman** [comment]
@kepano, can you make Logseq an official importer as part of the Obsidian import options -[https://obsidian.md/help/import](https://obsidian.md/help/import)? There are many users who prefer to stay with markdown and want to move from Logseq to Obsidian instead of Logseq DB.

**Comment by kepano** [comment]
> @kepano, can you make Logseq an official importer as part of the Obsidian import options -[https://obsidian.md/help/import](https://obsidian.md/help/import)? There are many users who prefer to stay with markdown and want to move from Logseq to Obsidian instead of Logseq DB.

Logseq is already listed there. We're welcoming PRs for Importer if someone wants to build one!

**Comment by fschuman** [comment]
> Logseq is already listed there. We're welcoming PRs for Importer if someone wants to build one!

Yes its listed under "Other Formats" for guides. I meant an official importer like for Notion or Evernote.

**Comment by kepano** [comment]
> > Logseq is already listed there. We're welcoming PRs for Importer if someone wants to build one!
> 
> Yes its listed under "Other Formats" for guides. I meant an official importer like for Notion or Evernote.

Both Importer and the Obsidian Help site welcome PRs. It would be ideal for someone who went through the migration to write that guide.

**Comment by laughedelic** [comment]
Hi everyone! I recently migrated (back) to Obsidian, and I decided to do it the proper way and implement an importer based on the official plugin. Here's my implementation: https://github.com/obsidianmd/obsidian-importer/pull/592

I kind of expect that this might not land well here because of the size of the changeset (too many tests, too much optional configuration), but I'm open to dialogue. If maintainers decide that this is not a good fit, I might release it as a separate plugin. But of course, I would prefer it to be part of the official plugin for all new users coming from Logseq (I think there will be more of them in the near future).

If anyone could also try it out on their notes and provide feedback, it would be great.
